src/app/settings/page.tsx

```diff
@@ -1,19 +1,20 @@
 "use client";
 
-import { Suspense, useCallback } from "react";
+import React, { Suspense, useCallback } from "react";
 import {
-  Button,
-  Container,
-  MenuItem,
-  Paper,
-  Select,
-  Switch,
-  Table,
-  TableBody,
-  TableCell,
-  TableContainer,
-  TableRow,
-  Typography,
+    Button,
```

Comment:
Make sure your editor is set up to use 2 spaces for indent (if your editor supports [EditorConfig](https://editorconfig.org/) it should pull in the settings automatically).

Reply:
It's weird because WebStorm is supposed to do that but for certain files it was set to 4 spaces...